events-api/src/modules/events/events.controller.ts

```diff
@@ -26,25 +29,51 @@ export class EventsController {
     const events = await this.eventsService.listEvents(searchParams);
 
     return {
+      message: 'Events succesfully received',
       data: events,
     };
   }
 
   @Get(':eventId')
   async getEventById(
-    @Param(
-      'eventId',
-      new ParseUUIDPipe({
-        errorHttpStatusCode: HttpStatus.BAD_REQUEST,
-        version: '4',
-      }),
-    )
+    @Param('eventId', ParsingUUIDPipe)
```

Comment:
Why it is called Parsing?

Reply:
Because this is an inherited class from ParseUUIDPipe @nestjs/common which is parses given value. I thought i will be right if a called it like this.

Now I rename it to UUIDValidationPipe